### PR TITLE
Add missing migrations for menu_name population from title

### PR DESCRIPTION
**Pull request by benbc**

### CODE REVIEW

File: reports/migrations/0016_populate_menu_name_from_title.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    dependencies = [
+        ("reports", "0015_report_contact_email"),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            code=populate_title_from_menu_name,
+            # There's no reasonable way to reverse this, but having the title populated is harmless.
+        ),
```
Comment (rebkwok) [inline reports/migrations/0016_populate_menu_name_from_title.py:26]: Does it run backwards OK without having `noop` explicitly set?

I usually make data migrations standalone, but I'm not sure I have a good reason for that, other than it helps me keep track of things to have them separate.  If it works forwards and backwards, seems fine to me

Comment (benbc) [reply reports/migrations/0016_populate_menu_name_from_title.py:26]: Oh, no! I misunderstood the API -- that will make it irreversible. I'll fix. Thanks.

Comment (benbc) [reply reports/migrations/0016_populate_menu_name_from_title.py:31]: Fixed.

File: reports/migrations/0016_populate_menu_name_from_title.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("reports", "0015_report_contact_email"),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            code=populate_title_from_menu_name,
+            reverse_code=noop,  # There's no reasonable way to reverse this, but having the title populated is harmless.
```
Comment (rebkwok) [inline reports/migrations/0016_populate_menu_name_from_title.py:30]: Sorry, should have been more explicit.  The usual way is to pass it `migrations.RunPython.noop`; so 
```
operations = [migrations.RunPython(populate_title_from_menu_name, reverse=migrations.RunPython.noop]
```
Then you don't need the custom `noop` function

**Review by rebkwok — APPROVED** [review]

**Pull request merged**